### PR TITLE
Added Rate Limits section

### PR DESCRIPTION
**Pull request by jmtremblay**
@alavers @jpjoyal @mspensieri 

- Added Rate Limits section
- Added 429 as possible error. 
- Added 409 as a bonus.

Question: do you think we should mention rate limiting on the SDKs pages too?

image

### CODE REVIEW

File: source/rest/index.md [hunk 1]
```diff
@@ -41,8 +41,14 @@ Smooch uses standard HTTP status codes to communicate errors
 | **401** | Unauthorized - Necessary credentials were either missing or invalid. |
 | **403** | Forbidden - Your credentials are valid but you don't have access to the requested resource. |
 | **404** | Not Found - The object you're requesting doesn't exist. |
+| **409** | Conflict - You might be trying to update the same resource concurrently. |
+| **429** | Too Many Requests - You are calling our APIs more frequently than we allow. |
 | **500, 502, 503, 504** | Server Errors - Something went wrong on our end. |
 
+## Rate Limits
+
+Smooch APIs are subject to rate limiting.  If you exceed the limits, Smooch will start returning a `429 Too Many Requests` HTTP status code. We apply rate limits to prevent abuse, spam, denial-of-service attacks, and similar issues. Our goal is to keep the limits high enough so that any application using Smooch as intended will never hit them. However, applications going consistantly over limits run the risk of being permantently disabled.
```
Comment (mspensieri) [inline source/rest/index.md]: consistantly -> consistently

Comment (mspensieri) [inline source/rest/index.md]: permantently -> permanently

**Comment by mspensieri** [comment]
Don't think it's worth it to mention on the sdk pages. The sdks have some measures in place to limit the amount of request they are allowed to send, and the `429` would be transparent to the developer anyways

File: source/rest/index.md (hunk 1 above)
Comment (jmtremblay) [inline source/rest/index.md]: Actually changing the last part to: "applications that consistently exceed limits run the risk..." as proposed by Stasi (I didn't save before commit..). Will fix "permanently" too.

**Comment by jpjoyal** [comment]
:+1:

**Comment by mspensieri** [comment]
:+1:

**Comment by alavers** [comment]
:+1:

**Pull request merged**